Maths/AverageMedian.js

```diff
@@ -8,7 +8,8 @@
  * else if the length of the array is odd number, the median value will be the middle number in the array
  */
 
-const averageMedian = (numbers) => {
+const averageMedian = (_numbers) => {
```

Comment:
This repo requires all variable names to be in lowerCamelCase and without underscores. Use different names for the variables.

Reply:
and without underscores - But why?

I may not have so much commercial experience, but wherever I have been, they used it.
This immediately makes it possible to distinguish input parameters from local variables and errors, like mutation input data, would most likely not occur

Reply:
Also easy work with autocomplete from ide. 
It almost never suggests incorrect values when autocomplete , unlike the suggested option.

Reply:
It is just a style followed by this repository, to make variable naming consistent. Also, as I previously mentioned, these algorithms do not need to be production ready of useful, just interesting and educative from a programming perspective.